mautic/mautic
# Activity feed block for Segment and Focus

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 


## Description

The activity feeds for the segment and focus items are missing.

| Before                                 | After
| -------------------------------------- | ---
| <img width="943" alt="segment - before" src="https://github.com/user-attachments/assets/e00c684f-0de6-4267-8dee-caf13166c632" /> |    <img width="937" alt="segment - after" src="https://github.com/user-attachments/assets/7085e90b-5ce2-4d55-9438-3dfbd48de688" />
|  <img width="942" alt="focus - before" src="https://github.com/user-attachments/assets/49dd7661-126a-4af6-acef-8c2d21e5589a" /> | <img width="941" alt="focus - after" src="https://github.com/user-attachments/assets/78beafdc-60b8-4929-bf62-620a3f920946" />


---
### 📋 Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create or update Segment and Focus Item
3. Navigate to the details page.
